internal/location/coordinates.go

```diff
@@ -0,0 +1,112 @@
+package location
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"math"
+	"math/rand"
+	"net/http"
+	"os"
+
+	mapbox "github.com/ryankurte/go-mapbox/lib"
+	"github.com/ryankurte/go-mapbox/lib/geocode"
+)
+
+const geocodeRequestLimit = 1
+const zipCodeTokenKey = "ZIP_CODE_TOKEN"
```

Comment:
Can we add this and any other new env variables (that you used for the postal code API) to Heroku. If you don't know how to do this we can take a look next work session

Reply:
I changed the token stuff to match how we did it for mapbox